xsd/netex_framework/netex_reusableComponents/netex_mode_support.xsd

```diff
@@ -436,4 +436,17 @@ Rail transport, Roads and Road transport
 			<xsd:enumeration value="other"/>
 		</xsd:restriction>
 	</xsd:simpleType>
+	<xsd:simpleType name="StopPlaceCategoryEnumeration">
```

Comment:
What's the difference between "Category" and "Type" - i.e., why has the name *StopPlace**Category**Enumeration* been chosen instead of *StopPlace**Type**Enumeration* (similar to the existing *Stop**Type**Enumeration* and other type names)?